core/baseline-core-v1.0-psd01.md

```diff
@@ -191,8 +191,273 @@ The ability of a Party operating Workflows on a baseline-compliant implementatio
+
+A peer-to-peer network is required as well as a consensus algorithm to ensure replication across nodes is undertaken [put reference here]
+
+For specificity, the popular word "Blockchain" is a particular form of DLT design.
+
+## 2.6 Baseline Protocol Instance
+
+Baseline Protocol Instances (BPIs) are logical constructs shared between (commercial) counterparties of Requesters and Providers and implemented on a Distributed Ledger. They are used to either validate, or reconcile commercial transactions between Requesters and Providers related to all (commercial) state objects transacted between them. The nature of bi- or multi-lateral transactions is such that two or more parties may (commercially) transact to/from each other interchangeably. 
+
+Abstractly, a BPI consists of
+* private messaging between agreement counterparties about the state or the requested or finalized state changes of the agreement between them,
+* the representation of a (commercial) agreement and its business rules and data as distinct workflows and worksteps between its counterparties organized into workgroups based on the stipulations of the (commercial) agreement
+* the deterministic processing and finalization of state change requests based on commercial documents between the agreement counterparties as stipulated by the (commercial) agreement.
+* the preservation of privacy of all agreement counterparties and their (commercial) data to other 3rd parties.
+
+BPIs are strongly dependent on the security and privacy capabilities of the DLT used to implement a BPI.
+
+**[R9]**	A BPI MUST be implemented on a DLT.
+
+Security and Privacy requirements of a BPI are key and are strongly dependent on the security and privacy assurances that the DLT on which the BPI is implemented can provide. BPIs need to take great care to avoid the following two situations:
+
+1. Weaken the security assurances of the underlying DLT by increasing the DLT attack surface. Such an expansion of the attack surface can occur through for example concentration of value-at-risk in one or more BPIs above the value used to economically secure the underlying DLT. This situation would make it economically attractive to attack, and subvert, the underlying DLT to extract the value in one or more BPIs.
+2.	Appreciably increase the existing attack surface of a DLT such that the security assurances of the BPI become significantly weaker than the underlying DLT. An example of such a situation can occur when a commercial state object such as a Financing contract or an Order in BPI A is dependent on a commercial state object such as an invoice as collateral in BPI B, and when BPI B has weaker transaction finality assurances than either BPI A or the underlying DLT. In that scenario, the commercial state object in BPI A cannot provably rely on the invoice as collateral in BPI B since the invoice might be reverted, and it would then no longer be a suitable collateral.
```

Comment:
Maybe 'Increase by an order of magnitude'?

Reply:
'State Object'

Reply:
Is this an example of frontrunning?

Reply:
I removed `appreciably` since it already qualified by `significantly weaker than the underlying DLT`. And no, not really, it is about how quickly the state on BPI B is finalized (say 120 seconds) compared to BPI A (instantly) and the DLT (say 90 seconds).